### PR TITLE
Changed www.sendle.com

### PR DESCRIPTION
**Pull request by michaelrob**
### The Problem

https://www.pivotaltracker.com/story/show/144910439

We are currently showing the api url as "www.sendle.com", however, this should show as "api.sendle.com".

### The Solution

Modified API documentation to show "https://api.sendle.com/api/..."
instead of "https://www.sendle.com/api/...".

### Review Requirements

This pull request only requires 1 reviewer.

### CODE REVIEW

**Comment by seangeo** [comment]
LGTM, @jstirk you cool with this as fitting on the pre-requisites for contacting API users?

File: source/includes/_getting_started.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ Any orders created on the sandbox server will be created in the test mode, they
 
 <aside class="notice">Stripe allows the use of <a href="https://stripe.com/docs/testing#cards"><strong>test credit card numbers</strong></a> which you can enter into the sandbox server to simulate credit card payments.</aside>
 
+<aside class="warning">For the sandbox, please make API calls to https://sandbox.sendle.com instead of https://api.sendle.com.</aside>
```
Comment (mendokusai) [inline source/includes/_getting_started.md]: I'm okay with it, but I wonder if *"sandbox"* will mean the same thing to everyone. Maybe it's better to specifically state testing, i.e. "For testing api calls, please use https://sandbox.sendle.com" and on second pass I suggest not including the actual api url for those who skip critical words :trollface:

Comment (michaelrob) [inline source/includes/_getting_started.md]: I'd be pretty happy with that!! @jstirk thoughts??

Comment (jstirk) [inline source/includes/_getting_started.md]: Good points @mendokusai .

"For testing the API, please make all API calls to https://sandbox.sendle.com." and then lets put this to bed for now.

**Pull request merged**